### PR TITLE
Replaces some MRE food sprites and descriptions

### PR DESCRIPTION
**Pull request by CosmicScientist**
All six normal meals from the canteen vending machines (not the protein bar) are resprited using tg sprites and given different names/descriptions to match.

Because why not.

### CODE REVIEW

**Comment by CosmicScientist** [comment]
dmca nufood

**Comment by Rohesie** [comment]
They are a bit uglier. Were the trays Nicboone's?

**Comment by CosmicScientist** [comment]
Dunno.

File: code/modules/cm_marines/equipment/gear.dm [hunk 1]
```diff
@@ -114,8 +114,8 @@
 	w_class = 2
 
 /obj/item/reagent_container/food/snacks/mre_pack/meal1
-	name = "\improper USCM Prepared Meal (cornbread)"
-	desc = "A tray of standard USCM food. Stale cornbread, tomato paste and some green goop fill this tray."
+	name = "\improper USCM Prepared Meal (bread)"
+	desc = "A staple of the Clown Planet, every clown baby's first solid food is a slice of banana bread with cream pie spread."
```
Comment (LetterN) [inline code/modules/cm_marines/equipment/gear.dm]: add creampies when the vendor is hacked

**Comment by Rohesie** [comment]
The issue here is that this is all cooked food. It looks horribly out of place and style in the food vendors.

If these sprites are really set to be replaced then food vendor stuff would make much more sense. Tin of beans perhaps, and stuff like that. Space food.

South of medbay there's room for a little kitchen for this kind of stuff.

File: code/modules/cm_marines/equipment/gear.dm (hunk 1 above)
Comment (CosmicScientist) [reply code/modules/cm_marines/equipment/gear.dm]: I'll see what I can do.

**Comment by CosmicScientist** [comment]
I'll see if I can find better sprites then.

**Comment by CosmicScientist** [comment]
banana bread tray
food tray
Making assets myself. Will have enough done soonish.

**Comment by CosmicScientist** [comment]
big banana bread tray
big meatballs tray
big monkey tray
big pizza tray
big pork and rice tray
big tofu tray
big tray

These any good before I touch the .dmis and git push?

**Comment by subject217** [comment]
>sopa de macacao 

uma delicia

**Comment by Rohesie** [comment]
Now these are great!

**Comment by CosmicScientist** [comment]
Alright that should be it unless someone else can point out a problem.

**Comment by LarissaMayrink** [comment]
>monkey mre
fucking love it

**Pull request merged**